tardis/interfaces/siteadapter.py

```diff
@@ -104,6 +105,13 @@ def drone_environment(
 
         return drone_environment
 
+    @property
+    def drone_heartbeat_interval(self) -> int:
+        try:
+            return self.site_configuration.drone_heartbeat_interval
+        except AttributeError:
+            return 60
```

Comment:
You are still returning a constant here. Is this still required? What about the default value you are instantiating in line 25 above?

Reply:
No, you are right. This is not needed anymore with pydantic. I will remove it.